spec/spec_helper.rb

```diff
@@ -28,23 +28,17 @@
     c.syntax = :expect
   end
 
-  config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
-  end
```

Comment:
This code was unnecessary because the database was re-created below.

https://github.com/kufu/activerecord-bitemporal/blob/e8f4263df3aeb0cf2166129f62fc55a29ad70b3e/spec/spec_helper.rb#L19-L22